deploy-manage/production-guidance/kibana-traffic-scaling-considerations.md

```diff
@@ -0,0 +1,102 @@
+
+Start {{kib}} on **2.1 vCPU** and **2GB** of memory. This should comfortably serve a set of 10 concurrent users performing analytics activities like browsing dashboards. If you are experiencing performance issues, adding an additional **2.1 vCPUs** and **2GB** per 10 concurrent users is a safe _minimum_ to ensure {{kib}} is not resource-starved for common analytics use cases. This is known as **vertical scaling** and should typically be employed up to a maximum of **8.4 vCPU** and **8GB** of memory. In combination, it is recommended to employ **horizontal scaling** as outlined in the guide on [load balancing traffic](./kibana-load-balance-traffic.md).
+
+Consider these examples:
+
+* {{kib}} to should serve 50 concurrent users: **10.5 vCPU** and **10GB** of memory which on {{ech}} and {{ece}} translates to: **2 {{kib}} instances of 8.4 vCPU and 8GB memory each** or **3 {{kib}} instances of 8.4 vCPU and 4GB memory each**
+* {{kib}} to serve 100 concurrent users you would need **25.2 vCPU** and **20GB** of memory which on {{ech}} and {{ece}} translates to: **3 {{kib}} instances of 8.4 vCPU and 8GB memory each**.
+
+::::{important}
+This advice does not apply to scaling {{kib}} for task manager. If you intend to use {{kib}} alerting capabilities see [task manager scaling guidance](./kibana-task-manager-scaling-considerations.md).
+::::
+
+**{{ece}}, {{ech}}, and {{eck}}** users can adjust {{kib}}'s memory by viewing their deployment and editing the {{kib}} instance's resource configuration. Note: size increments are predetermined.
+
+**Self-managed** users must provision memory to the host that {{kib}} is running on as well as configure allocated heap. See [the guidance on configuring {{kib}} memory](./kibana-configure-memory.md). **Note:** Node.js suggests allocating 80% of available host memory to heap, assuming that Kibana is the only server process running on the (virtual) host. This allows for memory resources to be used for other activities, for example: allowing for HTTP sockets to be allocated.
+
+**Serverless** manages {{kib}}'s resources automatically.
+
+### A more sophisticated sizing strategy
+
+::::{important}
+On [serverless](../deploy/elastic-cloud/serverless.md) scaling and configuring {{kib}} is fully managed for you.
+::::
+
+Building on the simple strategy outlined above, we can make more precise adjustments to resource allocations. **Self-managed** users manage their CPU and memory allocations independently and can employ the strategy below to further tailor resources based on performance metrics.
+
+#### Monitoring [_monitoring-kibana-metrics]
+
+In order to understand the impact of your usage patterns on **a single {{kib}} instance** use the Stack Monitoring feature. See [the guide for {{kib}} deployed on {{ech}} or {{ece}}](../monitor/stack-monitoring/ece-ech-stack-monitoring.md) or the [the guide for self-managed {{kib}}](../monitor/stack-monitoring/kibana-monitoring-self-managed.md).
```

Comment:
this also works for eck. would suggest just linking to the top level for stack monitoring.

also: could people use autoops for this? should they?

```suggestion
In order to understand the impact of your usage patterns on a single {{kib}} instance, use the [stack monitoring](/docs/deploy-manage/monitor/stack-monitoring) feature.
```